README.md

```diff
@@ -1 +1,275 @@
-# Pebble Bootstrap
+Slate
+=====
+
+Slate is a front-end framework for developing Pebble mobile configuration pages.
+It's the fastest way to make a clean UI for a Pebble app's mobile configuration
+page.
+
+*Current version: [v0.0.1][dist]*
+
+Getting Started
+---------------
+
+There are only two files that makeup the Slate framework, a CSS file and a
+JavaScript file.
+
+> Note that jQuery is also required, but not included in the framework.
+
+There are several quick ways to getting started with Slate.
+
+### Via CDN
+
+Pebble hosts both the CSS and JS files via CDN.
+
+```html
+<link href="https://somepebblecdn.com/slate/0.0.1/css/slate.min.css" rel="stylesheet">
+```
+
+```html
+<script src="https://somepebblecdn.com/slate/0.0.1/js/slate.min.js"></script>
+```
+
+### Via Download
+
+The CSS and JS files are also avaliable via download.
+
+[Download Slate 0.0.1 >](linktotherelease)
+
+### Via Bower
+
+The CSS and JS files are also avaliable via Bower.
+
+```bash
+bower install slate
+```
+
+Documentation
+-------------
+
+Here is a list of the different components you can create with Slate.
+
+### Paragraphs
+
+![paragraph](http://i.imgur.com/K7gXSGm.png)
```

Comment:
A lot of places block imgur. Is there somewhere better we can host these files? In the repo on the `gh-pages` branch maybe?